apps/src/music/analytics/AnalyticsReporter.ts

```diff
@@ -267,6 +236,19 @@ export default class AnalyticsReporter {
   }
 
   log(message: string) {
-    console.log(`[AMPLITUDE ANALYTICS EVENT]: ${message}`);
+    console.log(`[MUSIC AMPLITUDE ANALYTICS EVENT]: ${message}`);
+  }
+
+  private formatUserId(userId: number) {
```

Comment:
Copied from https://github.com/code-dot-org/code-dot-org/blob/f280b95523fb6a3aabbe624250146256a9480cef/apps/src/lib/util/AnalyticsReporter.js#L126 I'd imagine later we might want to consolidate/replace with this metrics reporter, so just copying over for now